tutorials/iot/ubuntu-kiosk/wayland-kiosk.md

```diff
@@ -304,23 +301,22 @@ apps:
 In our case, glmark2-wayland has the path `/usr/share/glmark2` hard-coded in, so this is not going to work for us.
 
 
-### Changing the application
+### 2. Changing the application
 
 Sometimes you can edit/recompile the application to add the environment variable mentioned above. If it is your own code you are snapping, this is a good approach.
 
 We could do this, but glmark2-wayland is not our own code and this adds an unnecessary maintenance overhead.
 
 
-### Using layouts for hard-coded paths
+### 3. Using layouts for hard-coded paths
 
-This experimental snapd feature bind-mounts directories inside the snap into any location, so that the binaries' hard-coded paths are correct. To use, add this to the YAML file:
+[This snapd feature](https://docs.snapcraft.io/snap-layouts) bind-mounts directories inside the snap into any location, so that the binaries' hard-coded paths are correct. To use, append this to the YAML file:
 
 
 ```yaml
-passthrough:
```

Comment:
I assume you've checked layouts work with the LP builders? Time has passed, but last time I tried it didn't work.

Reply:
Works now, I did a test:
https://code.launchpad.net/~gerboland/+snap/glmark2-example-test

Reply:
Works now, I did a test:
https://code.launchpad.net/~gerboland/+snap/glmark2-example-test